src/Visualization/Python/GenerateXdmf.py

```diff
@@ -243,10 +243,12 @@ def parse_args():
         formatter_class=ap.ArgumentDefaultsHelpFormatter)
     parser.add_argument(
         '--file-prefix',
+        '-i',
```

Comment:
I don't like `-i` being short for `file-prefix`, partly because the long form is not `--input`. The other concern is that I can reasonable see adding the ability to only parse a single (or an explicit list of) h5 file out of all of them (say if we know the elements of interest are all on node 0) then I'm not sure how to best name things.